package.json

```diff
@@ -53,5 +53,7 @@
 		"tslint-immutable": "4.7.0",
 		"typescript": "3.1.1"
 	},
-	"dependencies": {}
+	"dependencies": {
```

Comment:
u don't need to add the dependency here ='(

Reply:
Corrected the commit